src/config.json

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
There are issues with the structure of this file.

It should be:
```
"categories" : [
      {
          "name" :  "cat_name",
          "label" : "cat_label",
          "type" : "cat_type"
      }, ...
]
```

I know this might make it more difficult in respect to the `div` issue you were talking about but the structure of this file has to be very simple for the user.